activesupport/lib/active_support/subscriber.rb

```diff
@@ -40,6 +44,22 @@ def attach_to(namespace, subscriber = new, notifier = ActiveSupport::Notificatio
         end
       end
 
+      # Deattach the subscriber from a namespace.
+      def deattach_from(namespace, subscriber = new, notifier = ActiveSupport::Notifications)
```

Comment:
@sushantmittal Why is it `deattach` and not `detach`?

Reply:
@shailesh-kalamkar 

`deattach` is synonym of `detach` only.

Reply:
`detach` is definitely the more common spelling of the word.

Reply:
ok I will update it as we are using in rails already.